deno/payloads/v9/channel.ts

```diff
@@ -865,19 +865,45 @@ export interface APIFollowedChannel {
  */
 export interface APIReaction {
 	/**
-	 * Times this emoji has been used to react
+	 * Total number of times this emoji has been used to react (including super reacts)
 	 */
 	count: number;
+	/**
+	 * Reaction count details object
```

Comment:
This description just echoes the name of the interface, like it's as useful as not including it. Let's improve it:

```suggestion
	 * An object detailing the individual reaction counts for different types of reactions 
```